decidim-forms/app/forms/decidim/forms/answer_form.rb

```diff
@@ -44,6 +44,7 @@ def label
       def map_model(model)
         self.question_id = model.decidim_question_id
```

Comment:
This line is giving an error if you edit a form without all the questions answered in the first submit. 

Steps:

1. Fill a form with only the required questions answers. Leave empty the rest
2. Go to edit the form
3. See error

Reply:
fixed by 2621688129ba0b8f5565dd98e1438cd9b5325596